durable-storage/src/lib.rs

```diff
@@ -23,10 +23,12 @@
 //! - **Persistence layer**: Responsible for actually persisting the basic get, set, delete
 //!   operations on disk.
 
-#![cfg_attr(
+#[cfg_attr(not(test), expect(dead_code, reason = "Incomplete"))]
+mod merkle_layer;
+#[cfg_attr(
     not(test),
     expect(dead_code, reason = "Stubbed API - will be implemented in RV-793")
```

Comment:
Not directly related with this PR, but now the correct issue which will solve this lint is [RV-808](https://linear.app/tezos/issue/RV-808/database-gateway).
```suggestion
    expect(dead_code, reason = "Database will use the persistence layer in RV-808")
```

Reply:
Thanks, [done](https://github.com/tezos/riscv-pvm/compare/c7fe6ca937ada80c921db213d750935ecb94ad91..925048e322fd0aa30f904779f935961a85a47283#diff-43811d1a7735d058f37b579a10231f6513556249ce952ff6968fccddee8ad349R32)